contrib/node/tests/python/pants_test/contrib/node/tasks/BUILD

```diff
@@ -21,7 +34,7 @@ python_tests(
     'src/python/pants/fs',
     'src/python/pants/util:contextutil',
   ],
-  timeout=120,
+  timeout=300,
```

Comment:
When one timeout goes up, it's good to tighten a different one. Maybe you can bring each of `node_build` and `node_bundle` down to `timeout=30`?